components/Tabs/BUILD

```diff
@@ -173,18 +182,37 @@ mdc_objc_library(
     ],
 )
 
+mdc_objc_library(
+    name = "unit_test_tabbarview_sources",
+    testonly = 1,
+    srcs = native.glob([
+        "tests/unit/TabBarView/*.m",
+    ]),
+    sdk_frameworks = [
+        "XCTest",
+    ],
+    deps = [
+        ":TabBarView",
+    ],
+)
+
 mdc_unit_test_suite(
     name = "unit_tests",
     deps = [
         ":unit_test_sources",
+        ":unit_test_tabbarview_sources",
         ":unit_test_theming_sources",
     ],
 )
 
 mdc_snapshot_objc_library(
     name = "snapshot_test_lib",
-    extra_srcs = native.glob(["tests/snapshot/Theming/*.m"]),
+    extra_srcs = native.glob([
+        "tests/snapshot/TabBarView/*.m",
+        "tests/snapshot/Theming/*.m",
```

Comment:
Can this be removed? Do we need the Theming directory?

Reply:
Then the tests in `MDCTabBar_MaterialThemingSnapshotTests.m` won't be executed. Would you prefer two separate snapshot test targets - one for each extension?

Reply:
I would say it's a different component so a different BUILD target would be nice but it's up to you. I could be swayed either way.

Reply:
Ok. I just want this landed since it's blocking 2 other engineers doing the implementation. I'll rename it if that's what it takes.

Reply:
Done.